### DISCUSSION TITLE
Problem Parsing Package on any attempted update

### DISCUSSION THREAD

**Discussion by JonathanM32** [Navid200/xDrip#140] [Testing]
I'm running version 17cbcf0-2021.03.08 of xDrip on a Pixel 4 XL. I have been unable to update to any new version of xdrip because each time I do I got a pop up saying the throw is a problem parsing the package. It doesn't seem to matter what build it is. I'm running the latest version of Android 12

**Comment by Navid200** [comment]
Would it be possible to show a screenshot of the pop-up?

**Comment by Navid200** [comment]
Would you please post a screenshot of your classic system status page?

**Reply by JonathanM32** [reply]
![Screenshot_20211209-223529.png](https://user-images.githubusercontent.com/5178078/145513086-2b91f7ec-8cb9-497d-81dc-7faab5af645f.png)

![Screenshot_20211209-223503.png](https://user-images.githubusercontent.com/5178078/145513092-f96d2d4d-177d-4025-b67f-ce4733c9814c.png)

**Reply by Navid200** [reply]
Thanks.

Would you please open an issue (bug)?  https://github.com/NightscoutFoundation/xDrip/issues
Would you please include a link to this thread in the issue?
